Objects/descrobject.c

```diff
@@ -1963,14 +1963,19 @@ make_parameters(PyObject *args)
     return parameters;
 }
 
+_Py_IDENTIFIER(__parameters__);
```

Comment:
You could move this into the `if (alias->parameters == NULL)` block below (it expands to a `static` declaration).